spec/requests/site_controller_spec.rb

```diff
@@ -38,7 +38,7 @@
       get "/site/basic-info.json"
       json = response.parsed_body
 
-      expect(json.keys).not_to include("discourse_discover_enrolled")
+      expect(json.keys).not_to include("discourse_discover")
```

Comment:
No, now we include two more values inside "``discourse_discover`". So it will be like 
```json
"discourse_discover": {
  "enrolled": true,
  "logo_url": "",
  "locale": ""
}
```